sfge/src/test/java/com/salesforce/graph/build/MethodUtilTest.java

```diff
@@ -1189,55 +1189,6 @@ public void testIsTestClass() {
         MatcherAssert.assertThat(myTestClass.isTest(), equalTo(true));
     }
 
-    /**
-     * This only tests the positive UserClass case. The compiler does not accept annotations on the
-     * other types of vertices that are excluded from the query. Exclude test classes and methods
-     */
-    @Test
-    public void testGetAuraEnabledMethods() {
```

Comment:
For some reason, we have two `MethodUtilTest.java` files. This one is in `build` instead of `ops`, which led me to conclude that it's the one that should ultimately be deleted. But for now, I'm just moving the Aura-Enabled test into the other file.